pom.xml

```diff
@@ -1854,6 +1855,29 @@
         <version>${dep.commons-beanutils.version}</version>
       </dependency>
 
+      <!-- error prone -->
+      <dependency>
+        <groupId>com.google.errorprone</groupId>
+        <artifactId>error_prone_annotations</artifactId>
+        <version>${dep.error-prone.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.google.errorprone</groupId>
+        <artifactId>error_prone_core</artifactId>
+        <version>${dep.error-prone.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>jsr305</artifactId>
+          </exclusion>
+          <!-- errorprone javac isn't required on jdk 9+ with -XPlugin:ErrorProne -->
+          <exclusion>
```

Comment:
Not sure if we should include this exclusion.  It shouldn't be necessary unless you're trying to compile with jdk 7/8 and error-prone

Reply:
I think all of our use cases will probably be strictly JDK 11+, so I think this is a good thing to exclude.